docs/support/prometheus.md

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
I believe this file was added to `master` in 870d93e28f, so it's surprising to see it in this PR.. perhaps the branch could be rebased on the latest `master`?